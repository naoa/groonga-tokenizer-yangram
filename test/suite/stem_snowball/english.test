register tokenizers/yangram
yangram_register --stem_snowball en
tokenize TokenYaBigramStemSnowball "There are cars" NormalizerAuto
