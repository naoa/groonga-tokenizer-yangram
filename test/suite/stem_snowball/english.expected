register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register --stem_snowball en
[[0,0.0,0.0],"TokenYaBigramStemSnowball"]
tokenize TokenYaBigramStemSnowball "There are cars" NormalizerAuto
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "there",
      "position": 0
    },
    {
      "value": "are",
      "position": 1
    },
    {
      "value": "car",
      "position": 2
    }
  ]
]
